src/renderer/mesh_storage.zig

```diff
@@ -108,6 +109,9 @@ pub fn deinit() void {
 		mesh.decreaseRefCount();
 	}
 	priorityMeshUpdateList.deinit();
+	while(blockUpdateList.dequeue()) |blockUpdate| {
+		main.globalAllocator.free(blockUpdate.blockEntityData);
```

Comment:
IMO BlockUpdate should have deinit, it owns that memory and this ownership is a implementation detail.